inventory_management_system_api/schemas/catalogue_item.py

```diff
@@ -95,3 +95,8 @@ class CatalogueItemSchema(CreatedModifiedSchemaMixin, CatalogueItemPostSchema):
         description="The properties specific to this catalogue item as defined "
         "in the corresponding catalogue category"
     )
+
+    # Computed
+    number_of_spares: Optional[int] = Field(
+        default=None, description="Number of spares currently available within this catalogue item if known"
```

Comment:
```suggestion
        default=None, description="The number of spares currently available within this catalogue item"
```
All of the other descriptions start with `The` and `if known` doesn't seem necessary since other optional fields don't include this text.

Reply:
True, the others I have done have not as I thought it was unnecessary to have the `The`, but I will for the sake of keeping it the same in the file, the rest can be unified later. I think the `if known` is worth emphasising in this case? They others are directly edited by the user, so I expect if there is an optional `description` then they would understand that when its None it just doesn't have one rather than it not being known, whereas the spares it is specifically because it isn't known not just that it doesn't have one.

Reply:
Yeah I think it makes sense to leave `if known` in for this case. I thought there were some fields similar to `number_of_spares` (Optional, not known but exists), but I just realised I was looking at the Patch Schema where everything is made Optional